contentwarehouse/snippets/create_document_schema_test.py

```diff
@@ -0,0 +1,36 @@
+# # Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+
+from contentwarehouse.snippets import create_document_schema_sample
+from contentwarehouse.snippets import test_utilities
+
+import pytest
+
+project_id = os.environ["GOOGLE_CLOUD_PROJECT"]
+location = "us"
+
+
+def test_create_document_schema(capsys: pytest.CaptureFixture) -> None:
+    project_number = test_utilities.get_project_number(project_id)
+
+    create_document_schema_sample.sample_create_document_schema(
+        project_number=project_number,
+        location=location)
+
+    out, _ = capsys.readouterr()
+
+    assert "Document Schema Created" in out
```

Comment:
Replace with an assertion on the return value. From the documentation, I assume that any failure of the API will throw an exception, so any return value that isn't None indicates success.

Reply:
The created schema will continue to exist after this test, and the number of unused schemas will grow every day from CI tests. So the test should also clean up after itself and delete the schema it just created.

Suggestion: combine the tests for creating and deleting into one test.